ui/src/app/views/data-use-agreement.tsx

```diff
@@ -233,28 +248,86 @@ const InitialsAgreement = (props) => {
   </div>;
 };
 
-const sanctionsDocUrl =
-  'https://docs.google.com/document/d/1O9PLBggXMpL5ri930sV-uDQ55J3Et9Ry31gdyrN7dVA/edit#';
+const SanctionModal = (props) => {
+  return <Modal width = {750}>
+    <ModalTitle style={styles.sanctionModalTitle}>
+      All of Us Research Program - Sanctions on Violations of the Code of Conduct
+    </ModalTitle>
+    <ModalBody>
+      <label style={styles.modalLabel}>
+        The Resource Access Board (RAB) of the <i>All of Us</i> Research Program determines whether
+        an investigator has violated the Code of Conduct outlined in the Data Use Agreement
+        signed by each user. The RAB notifies the All of Us Research Program office of the
+        violation.
+        <IndentedUl>
+          <li>
+            <label>The <i>All of Us</i> Research Program office and/or the All of Us IRB may
+            implement the following sanctions if it is determined that an investigator has violated
+            the Code of Conduct:
+            </label>
+            <IndentedUl>
+              <IndentedLi>
+                Determine whether any action by the investigator is required to remedy the
+                violation.
+              </IndentedLi>
+              <IndentedLi>
+                Determine whether any action by the investigator is required to remedy the
+                violation.
+              </IndentedLi>
+              <IndentedLi>
+                Revoke and/or deny access of the violator to all non-public (Registered and
+                Controlled tier) <i>All of Us</i> data.
+              </IndentedLi>
+              <IndentedLi>
+                Post the name and affiliation of the violator on a public <i>All of Us</i>
+                Research Program webpage.
+              </IndentedLi>
+              <IndentedLi>
+                Revoke extant NIH funding and/or prohibit future funding, either permanently or for
+                an explicit period of time
+              </IndentedLi>
+              <IndentedLi>
+                Prosecute the violator for breach of a contract with the federal government
+              </IndentedLi>
+            </IndentedUl>
+          </li>
+        </IndentedUl>
+      </label>
+    </ModalBody>
+    <ModalFooter>
+      <Button type='primary' onClick={props.onClose}>CLOSE</Button>
```

Comment:
button will automatically capitalize; "Close" does not need to be in caps (and shouldn't, in case we decide to change the button format)